src/main/java/io/appform/jsonrules/expressions/JsonPathBasedExpression.java

```diff
@@ -39,21 +39,26 @@
     private String path;
     private PreOperation<?> preoperation;
     private static final ObjectMapper mapper = new ObjectMapper();
+    private Boolean defaultResult;
```

Comment:
Hey default value would not always be false. Check the behaviour of negative operators.
not_equals operator for path not present currently evaluates to true.

_Sent from my OnePlus ONEPLUS A5000 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_